provider/resources.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mssql
+
+import (
+	_ "embed"
+	"path/filepath"
+	"strings"
+
+	shimprovider "github.com/PGSSoft/terraform-provider-mssql/shim"
+	"github.com/ettle/strcase"
+	pf "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
+	"github.com/pulumiverse/pulumi-mssql/provider/pkg/version"
+)
+
+//go:embed cmd/pulumi-resource-mssql/bridge-metadata.json
+var bridgeMetadata []byte
+
+// all of the token components used below.
+const (
+	// modules:
+	mainMod = "index" // the mssql module
```

Comment:
Not a comment, rather a question: do you want all your resources and `get`-functions to end up in the top level module of every generated SDK? By leveraging other module names, you can group certain resources in submodules.

Example in `resources.go`:
https://github.com/pulumiverse/pulumi-unifi/blob/8240d3d4992fc76fc1676b71132c86f2be06c6ed/provider/resources.go#L38-L42

How this reflect in the SDKs & API docs:
https://www.pulumi.com/registry/packages/unifi/api-docs/

Reply:
A fair question. However, the provider is so small that I don't think it's worth creating different modules.
![image](https://github.com/pulumiverse/pulumi-mssql/assets/14177833/1efa5423-3b4c-4466-9a8b-7f68548440d0)

Reply:
👍🏻